bin/tictactoe

```diff
@@ -1,3 +1,61 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Tic Tac Toe!"
```

Comment:
I see what you mean! I made a CLI class and did some refactoring so it's more readable.